app/uk/gov/hmrc/exports/services/WCOPointerMappingService.scala

```diff
@@ -16,27 +16,47 @@
 
 package uk.gov.hmrc.exports.services
 
-import javax.inject.Inject
+import com.github.tototoshi.csv.CSVReader
+import play.api.Logger
 import uk.gov.hmrc.exports.models.{Pointer, PointerMapping, PointerPattern}
-import uk.gov.hmrc.exports.util.FileReader
-
-class WCOPointerMappingService @Inject()(fileReader: FileReader) {
-
-  private lazy val mappings: Set[PointerMapping] = {
-    fileReader
-      .readLines("code-lists/pointer-mapping.csv")
-      .map(_.split(","))
-      .map {
-        case Array(wcoPattern, exportsPattern) =>
-          PointerMapping(PointerPattern(wcoPattern.trim), PointerPattern(exportsPattern.trim))
-      }
-      .toSet
+
+import scala.io.Source
+
+object WCOPointerMappingService {
```

Comment:
Why the change away from class with injected file reader?

Reply:
I had a problem with Injected one, Null Pointers.

Additionally I decided to use one approach, we used this approach in frontend application.

It contains static content, which is loaded during application startup, so I think object is okay